src/main/java/com/example/demo/console/ConsoleInOut.java

```diff
@@ -0,0 +1,94 @@
+package com.example.demo.console;
+
+import com.example.demo.dao.MatchDAO;
+import com.example.demo.model.Match;
+import com.example.demo.model.Team;
+import com.example.demo.tournament.Tournament;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Scanner;
+
+@Component
+@AllArgsConstructor
+public class ConsoleInOut {
+    @Getter
+    private final Tournament tournament;
+    private final Scanner in;
+
+
+    public void outTeam(Team team){
```

Comment:
Better to name in this way or similar: `printTeams` / `printTeamsNames` / `showTeamsNames` / etc...